ONBOARDING.md

```diff
@@ -0,0 +1,138 @@
+## Getting Started
+
+### Core SDK
+
+First access the mParticle Core SDK repository via the following
+link: [mParticle GitHub Repo](https://github.com/mParticle/mparticle-android-sdk)
+You can easily "fork" the repository by clicking the **Fork** tab in the upper right corner, once
+that is done you should be able to see it in your own Github account under the **Repositories** tab.
+Open your forked version and in the green **Code** tab copy the https link. Go ahead and open
+Android Studio and select Project from Version Control or Get from VCS, paste the Repository link
+you just copied and select **Clone**.
+
+### Testing the Core SDK
+
+Once the project is loaded, run the default tests to validate that your environment is set up
+correctly and to check that the code is working correctly before making changes of your own.
+
+You can do so locally using the the following gradle wrapper commands in your terminal:
+
+For Lint Checks:
+`./gradlew lint`
+
+For Unit Tests:
+`./gradlew test`
+
+For Instrumented Tests:
+`./gradlew :android-core:cAT :android-kit-base:cAT --stacktrace`
+
+In some machines running windows you may need to type the commands without **./**
+eg. `gradlew lint`
+
+#### Issues with JAVA_HOME
+
+When running the tests for the first time, It is possible that you run into the following issue:
+
+**ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.**
+
+**Please set the JAVA_HOME variable in your environment to match the location of your Java
+installation.**
+
+Any version between Java 8 and 11 is required to execute gradle, **Any Java versions above that are
+not supported yet.** [Latest Release Notes](https://developer.android.com/studio/releases)
+Make sure that you have installed a JDK and the path **JAVA_HOME** is created in the system
+Environment Variables.
+
+To install the JDK software, do the following:
+Go to [Official Oracle JDK download Website](https://www.oracle.com/java/technologies/downloads/)
+select the appropriate JDK software and click Download. The JDK software is installed on your
+computer in the default location; for example, at C:\Program Files\Java\jdk1.6.0_02.
+
+To set JAVA_HOME, do the following:
```

Comment:
specify "For Windows users", instructions for Mac Linux are different.

For mac it would be like:
"Open the '.bash_profile' file and add the following line: `export PATH="{PATH_TO_MY_JAVA}:$PATH", where PATH_TO_MY_JAVA is the file path for you JDK installation"